### DISCUSSION TITLE
What skill development, training or career opportunities do we need? (Session one, room two)

### DISCUSSION THREAD

**Discussion by junranlei** [RSEAA/RSEAA.github.io#141] [Discussion topics for #RSEAA24]
Discussion can include (please feel free to make other suggestions too):
What skill development, training or career opportunities do we need?
How do we get these currently and how can the processes be improved?
Is there funding to support these? How do we apply?
Suggestions for the formats, delivery, assessment...?

**Comment by lindaerlina** [comment]
At Universitas Indonesia, we have funding for research but sometimes the cost of doing the wet lab things is so high, the problem is we need to seek another funding to cover the publication cost.

**Comment by kozo2** [comment]
> What skill development, training or career opportunities do we need?

I believe that we need to spread the system of The Carpentries in our country.
https://carpentries.org/index.html

> How do we get these currently and how can the processes be improved?

To get in it, we just need to join the community.

> Is there funding to support these? How do we apply?

There are some fundings for it. (For example https://chanzuckerberg.com/eoss/proposals/bioconductor-high-quality-training-and-support-for-a-worldwide-community/)
We can apply it from https://chanzuckerberg.com/eoss/proposals/.

> Suggestions for the formats, delivery, assessment...?

I think the Carpentries's format and delivery are good.
(See `Upcoming Carpentries Workshops` in https://carpentries.org/index.html .
(On the other hand,) I don't know about the `assessment` in the Carpentries.

**Reply by junranlei** [reply]
The Carpentries is a great idea! Thanks for sharing the information.

**Comment by junranlei** [comment]
Safaet Hossain shared this webinar link during the discussion: https://youtu.be/CQ6op_LoHS0 
Some notes/summary from the session: There's interest in organising collaborative webinars on AI, machine learning, software engineering or other emerging topics that can engage participants from different countries and institutions. These webinars would be open to the public and recorded for future access on platforms like YouTube. There is also mention of international collaboration, such as partnerships with universities in the US, Indonesia, and others, to facilitate knowledge exchange through webinars and guest lectures. Previous webinars indicate the broad reach and interest in these activities. The idea of seeking international funding to support these initiatives is also discussed, along with the benefits of sharing expertise and experiences globally.

**Comment by junranlei** [comment]
The information shared by Rowland in the main room is interesting:
https://github.com/RSEAA/RSEAA.github.io/discussions/131#discussioncomment-10609943
